SPEC/FILES.md

```diff
@@ -181,17 +181,45 @@ If no `callback` is passed, a promise is returned.
 
 **Example:**
 
+In the browser, assuming `ipfs = new Ipfs(...)`:
+
+```js
+let content = ipfs.types.Buffer.from('ABC');
+let results = await ipfs.files.add(content);
+let hash = results[0].hash; // "Qm...WW"
+```
+
+Now [ipfs.io/ipfs/Qm...WW](https://ipfs.io/ipfs/QmNz1UBzpdd4HfZ3qir3aPiRdX5a93XwTuDNyXRc6PKhWW)
+returns the "ABC" string.
+
+The following code creates a directory object:
```

Comment:
I think we just need to tweak this wording to something like: "The following allows you to add multiple files at once. Note that intermediate directories in file paths will be automatically created and returned in the response along with files:"